benchmarks/nemo/nemolite2d/psykal/Makefile

```diff
@@ -125,11 +132,35 @@ nemolite2d_ocl: common_lib timer_lib serial_inf_lib fortcl_lib nemolite2d_alg.f9
 	@echo "OpenCL version prepared."
 	@echo "Run 'make <target>' inside the $@ folder"
 
+# MPI+OpenCL PSyclone code-generation of NemoLite2D
+nemolite2d_mpiocl: parallel_libs fortcl_lib nemolite2d_alg.f90 psyclone_scripts/ocl_transform.py
+	@echo "Generating NemoLite2D with OpenCL"; rm -rf $@; mkdir -p $@
+	# This PSyclone invokation will need to resolve some Fortran modules which need
+	# to be found in the path provided by the -I parameter.
+	${PSYCLONE} -s psyclone_scripts/ocl_transform.py --kernel-renaming single \
+		-I ${COMMON_DIR} \
+		-okern $@ -oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
+	cp ${INF_LIB} $@/. # Each generated folder needs its purposely built inf_lib
+	cp namelist $@/.
+	cp Makefile_gen $@/Makefile
+	# Currently we need all kernels in a single OpenCL file
+	cat $@/*.cl >> $@/allkernels
```

Comment:
Oops, I think I was just worried of the output file being found at the wildcard expansion, but it will not happen as the expansion is evaluated before the write.